### ISSUE TITLE
"ERROR:  Description:  Unknow Error" when building pkg on Sierra with valid Developer ID Installer cert

### ISSUE THREAD

**Issue by Lomanic** [packagesdev/packages#3]
Following the [CLI documentation](http://s.sudre.free.fr/Software/documentation/Packages/en/Project_Building.html#4)

```
packagesbuild -v Redacted.pkgproj
Building Project (14:29:14) at path: /Users/[Redacted]/Redacted.pkgproj 
------------------------------------------------------------------------------

Build Folder (done)

Distribution

	Package "Redacted"

		Payload

			Assemble file hierarchy (done)
			Split forks (done)
			Create bill of material (done)
			Create pax archive (done)
		
		Scripts

			Assemble file hierarchy (done)
			Split forks (done)
			Create pax archive (done)
		
		PackageInfo (done)

	----------------------------------------------

	Package "Redacted2"

		Payload

			Assemble file hierarchy (done)
			Split forks (done)
			Create bill of material (done)
			Create pax archive (done)
		
		Scripts

			Assemble file hierarchy (done)
			Split forks (done)
			Create pax archive (done)
		
		PackageInfo (done)

	----------------------------------------------

	Definition

		Installation|Volume requirements (done)

		License (done)

		Choices hierarchies (done)

		JavaScript scripts (done)

	Resources (done)

	Plugins (done)


==============================================================================
ERROR:

Description:

Unknow Error

==============================================================================
Build Failed
```

After removing the developer certificate I can build it properly: this was hinted by the following tweet https://twitter.com/mscottblake/status/793899294852087809

Is there a workaround to call `productsign` after a successful pkg build from the GUI? I can currently do the following of course:

``` bash
packagesbuild -v Redacted.pkgproj && productsign --sign 'Developer ID Installer: REDACTED' build/Redacted.pkg build/Redacted-signed.pkg && mv build/Redacted-signed.pkg build/Redacted.pkg
```

**Comment by rudyrichter** [comment]
I don't believe so, it will have to wait for the major update that @packagesdev is working on.

See: http://s.sudre.free.fr/Software/Packages/Q&A_6.html

**Comment by packagesdev** [comment]
This is solved with version 1.2 that is now available.

http://s.sudre.free.fr/Software/Packages/release_notes.html
